src/appstartdialog.cpp

```diff
@@ -154,6 +154,10 @@ void mmAppStartDialog::CreateControls()
     {
         mmToolTip(itemButton61, wxString::Format(_("Open the previously opened database : %s"), val));
     }
+    wxButton* ok = static_cast<wxButton*>(FindWindow(wxID_OK));
```

Comment:
This fix does not make sense, because the corresponding code is already a little higher.

```
    m_buttonClose = new wxButton(this, wxID_OK, _("&OK "));
    m_buttonExit = new wxButton(this, wxID_EXIT, _("E&xit "));
```

Reply:
Agreed. This is not the issue. Its here https://github.com/moneymanagerex/moneymanagerex/blob/master/src/appstartdialog.cpp#L212 where wxGetSingleChoice doesn't translate 'OK'.